### DISCUSSION TITLE
Where can I find the tree-sitter playground's index.html?

### DISCUSSION THREAD

**Discussion by ahelwer** [tree-sitter/tree-sitter#3259] [Q&A] [answered]
For hosting my own demo website, as in https://github.com/tlaplus-community/tlaplus-community.github.io

I updated the web-tree-sitter and WASM components and now it doesn't work. I even found the playground.js file in this repo. However, I suspect I need to upgrade the index.html as well.

I also recently began building & distributing my grammar's WASM with emscripten 3.x instead of 2.0.16. This might have broken things as well.

**Comment by amaanq** [comment, marked as answer]
https://github.com/tree-sitter/tree-sitter/blob/master/cli/src/playground.html & https://github.com/tree-sitter/tree-sitter/blob/master/docs/assets/js/playground.js

**Reply by ahelwer** [reply]
Thanks! Have there been any other reports of playground websites breaking when upgrading to emscripten 3.x or upgrading to a newer version of web-tree-sitter? Or am I the only one who actually runs these things in my own website lol

**Reply by amaanq** [reply]
Hm, what version exactly? I build the wasm files with 3.1.55 and they're running fine on the site. Your issue might be using an incorrect `tree-sitter.js` file, try running `./script/build-wasm` in the repo and using the js/wasm file in `lib/binding_web`
